tests/echo_service_tests.rs

```diff
@@ -0,0 +1,40 @@
+use rusk::services::echoer::basic_proto::echoer_client::EchoerClient;
+use rusk::services::echoer::basic_proto::echoer_server::EchoerServer;
+use rusk::services::echoer::basic_proto::EchoRequest;
```

Comment:
Maybe we could redesign the modules here or re-export the basic_proto. So basically I would expect something like:
```rust
use rusk::services::echoer::{ 
  EchoerClient, EchoerServer, EchoerRequest
};
```

Reply:
Done, I don't quite like to export the client since will only  be used by the tests. But it's fine for the time being. Adressed.